src/react-three-hook/default-threejs-setup.ts

```diff
@@ -1,16 +1,12 @@
 import * as THREE from "three";
 
 export const getDefCamera = ({ offsetWidth, offsetHeight }: { offsetWidth: number, offsetHeight: number }) => {
-  const camera = new THREE.PerspectiveCamera(
+  return new THREE.PerspectiveCamera(
```

Comment:
Minor thought - not sure if we'll need to adjust these settings at all but maybe in future there will be zoom in-out that might also adjust the FOV - I guess we would be more likely to simply move the camera but sometimes a move + FOV tweak is nice. Initial settings for FOV and near/far settings could be pulled out as constants and potentially used as constraints.

Reply:
These properties are accessible as camera props and can be updated at any time. Actually, it's all handled by THREE.OrbitControls. I'd need to check what they do when you zoom in/out (we can do that now). My guess is that they just move the camera away.

btw, I thought that the Orthographic camera might be actually better in this model. We don't really need perspective here. E.g. spark labels are getting a bit smaller when you place them on the far edge of the model. :wink: Having orto cam we could even make them match cursor perfectly. Some other calculations might be easier too.